### PR TITLE
Add example usage of the second parameter of the onSubmit method.

### PR DESCRIPTION
**Pull request by beausmith**
More docs = more knowledge!

Closes: https://github.com/jaredpalmer/formik/issues/1107

Related: https://github.com/jaredpalmer/formik/issues/234

### CODE REVIEW

**Comment by jaredpalmer** [comment]
Cool. Just a heads up that `FormikActions` type will be renamed to `FormikHelpers` in v2.

File: docs/guides/typescript.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const MyApp: React.SFC<{}> = () => {
       <h1>My Example</h1>
       <Formik
         initialValues={{ firstName: '' }}
-        onSubmit={(values: MyFormValues) => alert(JSON.stringify(values))}
+        onSubmit={(values: MyFormValues, actions: FormikActions<MyFormValues>) => console.log({ values, actions })}
```
Comment (jaredpalmer) [inline docs/guides/typescript.md]:
```suggestion
        onSubmit={(values: MyFormValues, actions: FormikActions<MyFormValues>) => {
            console.log({ values, actions });
            alert(JSON.stringify(values, null, 2));
            actions.setSubmitting(false)
         }}
```

Comment (jaredpalmer) [reply docs/guides/typescript.md]: We should actually use the actions. Also, as of Formik 1.4.3 all of these types get inferred so you actually don't need these at all. 🤷‍♂️

**Comment by beausmith** [comment]
@jaredpalmer - I'm stoked to hear that the types are supposed to be inferred in Formik 1.4.3. 

I am using 1.4.3 with VS Code (with the built-in `ms-vscode.vscode-typescript-tslint-plugin` plugin) and it's complaining…

image

image

I'd love to be able to have the types be inferred! Any tips to fix this?

**Comment by jaredpalmer** [comment]
Ahh I meant they are inferred if you declare the function inline

**Pull request merged**